spec/services/provision_token_spec.rb

```diff
@@ -17,6 +26,22 @@
       expect(decoded_token).to_not be_nil
     end
 
+    it "sets the generated token's client name" do
+      expect(decoded_token.first['client']).to eq('foobar')
+    end
+
+    it "sets the generated token's permissions" do
+      expect(decoded_token.first['access']).to eq(permissions_string.split)
+    end
+
+    it "sets the generated token's issued at (iat)" do
+      expect(decoded_token.first['iat']).to eq(Time.now.to_i)
```

Comment:
This might fail in some cases.
Could we make it more robust using something like `be_within(1.second).of Time.now` ?